account_invoice_warn_message/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+- Héctor Villarreal \<<hector.villarreal@forgeflow.com>\>
+- Manuel Regidor \<<manuel.regidor@sygel.es>\>
```

Comment:
Use this format please:

https://github.com/OCA/l10n-spain/blob/6c9a473b1607f52da7295fafaaae4eae70149f32/l10n_es_aeat_mod303/readme/CONTRIBUTORS.md?plain=1#L9-L11